src/net.h

```diff
@@ -848,6 +848,8 @@ class CNode
             }
         } else if (inv.type == MSG_BLOCK) {
             vInventoryBlockToSend.push_back(inv.hash);
+        } else {
+            LogPrint(BCLog::NET, "%s: Unknown CInv type:%s\n", __func__, inv.ToString());
```

Comment:
This is the only log call, should include loggin.h if this is accepted.

Reply:
It's including LogPrint in logging.h indirectly via system.h, for example addrman.h > system.h > logging.h.

Reply:
We tend to https://include-what-you-use.org/.

Reply:
Good to know, thanks.